EC-CUBE/ec-cube
# 【マイページ】赤伝(マイナス数量の商品)を含む受注は、再注文ボタンを「非表示」にする

## 概要(Overview・Refs Issue)

  - if (orderにマイナス数量の商品明細が含まれている)
  - 再注文ボタンを「非表示」にする 
     こんな感じです

## 方針(Policy)



## 実装に関する補足(Appendix)


## テスト（Test)

 - Make order in front-end
 - In Admin, change quantity to minus
 - Check Order button have to invisible 

## 相談（Discussion）


## マイナーバージョン互換性保持のための制限事項チェックリスト


- [x] 既存機能の仕様変更
- [x] フックポイントの呼び出しタイミングの変更
- [x] フックポイントのパラメータの削除・データ型の変更
- [x] twigファイルに渡しているパラメータの削除・データ型の変更
- [x] Serviceクラスの公開関数の、引数の削除・データ型の変更
- [x] 入出力ファイル(CSVなど)のフォーマット変更
